.github/workflows/pr.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     secrets: inherit
 
   test:
-    name: Unit tests
+    name: Tests
```

Comment:
![Screenshot 2024-05-17 at 3 02 56 PM](https://github.com/calcom/cal.com/assets/2538462/ed4f527d-af59-4d78-8261-ef7904a6980d)

Naming all of these `Tests` results in the following grouping, which is nicer.